.github/workflows/stale.yml

```diff
@@ -0,0 +1,18 @@
+name: "Detect stale issues"
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: "This issue is marked as stale because it has been open 90 days with no activity."
+        stale-issue-label: "stale"
+        stale-pr-message: "This PR is marked as stale because it has been open 90 days with no activity."
+        stale-pr-label: "stale"
+        days-before-stale: 90
+        days-before-close: -1
```

Comment:
This disables auto-close. Not sure if we want to enable it.

Reply:
Closing stale PRs manually is acceptable. In this case human interaction is acceptable or even desired.